empress/reconcile/recon_vis/recon_viewer.py

```diff
@@ -17,7 +17,8 @@ def render(host_dict, parasite_dict, recon_dict, show_internal_labels=False, sho
     :param parasite_dict:  Parasite tree represented in dictionary format
     :param recon_dict: Reconciliation represented in dictionary format
     """
-    host_tree, parasite_tree, recon = utils.convert_to_objects(host_dict, parasite_dict, recon_dict)
+    # TODO: consistency_type is not used
```

Comment:
I feel like we should not put todos in the code because it can easily be forgotten. I think what we can do is either create an issue and assign Prof Ran to the issue and then post the link to the issue under the todo, or remove this `# TODO:`.